src/main/java/com/example/payroll/Repositories/EmployeesRepository.java

```diff
@@ -0,0 +1,7 @@
+package com.example.payroll.Repositories;
+
+import com.example.payroll.Models.Employees;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface EmployeesRepository extends JpaRepository<Employees, Long> {
```

Comment:
If we will rename entity to `Employee`
Then the repo should also be renamed

Reply:
done